app/views/users/_post.html.erb

```diff
@@ -1,4 +1,4 @@
-<%- @user.posts.each do |post| -%>
+<%- @user.three_most_recent_post.each do |post| -%>
     <div class="user_details_post_container">
```

Comment:
- [ ] [OPTIONAL] An error occurs when rendering the three most recent posts on the `user#show` page. This error is caused due to `three_most_recent_post` being undefined. I think you meant `three_most_recent_posts`.
### NOTE: This is optional because the previous reviewer missed it. 
![image](https://user-images.githubusercontent.com/46839548/232244884-1bf5cb03-77a8-4b2a-b399-7eaeefe83053.png)

Reply:
Reconciled. Thanks